src/format/strftime.rs

```diff
@@ -209,7 +209,7 @@ static D_T_FMT: &[Item<'static>] = &[
 static T_FMT: &[Item<'static>] = &[num0!(Hour), lit!(":"), num0!(Minute), lit!(":"), num0!(Second)];
 
 /// Parsing iterator for `strftime`-like format strings.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Copy)]
```

Comment:
Hmm, why did you decide to make this `Copy`? I guess it's technically qualifies, but it seems to be on the larger side to make copying implicit.

Reply:
This was done because the check I add in `src/datetime/mod.rs` will take ownership of the `StrftimeItems` preventing it being passed into `format_with_items` and this was the simplest way to solve the problem.
If you think this isn't appropriate I can take a different approach.

Reply:
Hm, yeah. I think the right way to get here is to remove the `Item::Error` variant and then fix it up so that everything that creates an `Item` becomes fallible instead.

Reply:
Wait, this code is weirder than I remembered. So it feels to me like `StrftimeItems` should probably contain a single `&[Item<'_>]`, with `new()` eagerly parsing it's `&str` argument. But I don't know how big of a change that is...